website/docs/httpsonly.md

```diff
@@ -44,20 +54,47 @@ spec:
           re_match("^(extensions|networking.k8s.io)/", input.review.object.apiVersion)
           ingress := input.review.object
           not https_complete(ingress)
+          not tls_is_optional(ingress)
           msg := sprintf("Ingress should be https. tls configuration and allow-http=false annotation are required for %v", [ingress.metadata.name])
         }
 
+        violation[{"msg": msg}] {
```

Comment:
this change is due to this policy getting updated but not docs. opened #232 so we require regen